modules/mostrarFechaHora.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+const mostrarFechaHora = () => {
```

Comment:
- [ ] As mentioned by the previous reviewer and for best practices and clean code, kindly ensure that you fix your linters errors instead of disabling them manually from your codebase.

Reply:
You are checking wrong I don't have any module with that name

Reply:
I hope an answer please

Reply:
Hi @AndresEspi 

Wait a second. I am on it

Reply:
Hi @AndresEspi 

I think it was an issue with github as I commented them on your pull request. But right now I am checking they are not there.

Reply:
Do I have to resubmit it to another reviewer?

Reply:
I think so. The issue was coming from Github.